dub.selections.json

```diff
@@ -1,13 +0,0 @@
-{
-	"fileVersion": 1,
-	"versions": {
-		"dyaml": "0.5.3",
-		"tinyendian": "0.1.2",
-		"libasync": "0.7.9",
-		"memutils": "0.4.5",
-		"openssl": "1.1.4+1.0.1g",
-		"vibe-d": "0.7.28",
-		"libevent": "2.0.1+2.0.16",
-		"libev": "5.0.0+4.04"
-	}
-}
```

Comment:
Unfortunately git rm in this case deleted the file :-) git rm is okay when you want to remove *staged* changes. In this case `git checkout HEAD -- dub.selections.json` should reset the file to the normal state without removing it..